operator/opensearchcontroller/webhook.go

```diff
@@ -114,7 +112,18 @@ func (v *Validator) compareZone(old, new *exoscalev1.OpenSearch) error {
 }
 
 func (v *Validator) compareVersion(old, new *exoscalev1.OpenSearch) error {
```

Comment:
I believe we should not allow to change the version therefore I would make this field immutable. Changing the version would imply that the customer would wrongly believe that he can change the version of his running instance. @Kidswiss do you agree with that?
P.S: Applies to all DBaaS